packages/core/src/components/DateRangePicker/DateRangeCalendar/Month/Month.tsx

```diff
@@ -59,7 +59,7 @@ export const Month: React.FC<Props> = React.memo(
                             isWeekFirstDate = isSameDay(_date, startOfWeek(_date)),
                             isWeekLastDate = isSameDay(_date, endOfWeek(_date)),
                             isHoverDateAfterDateSelection = !!(startDate && !endDate) || !!(!startDate && endDate),
-                            disabled = _date > maxSelectableDate || _date < minSelectableDate;
+                            disabled = endOfDay(_date) > endOfDay(maxSelectableDate) || endOfDay(_date) < endOfDay(minSelectableDate);
```

Comment:
I guess ``disabled = _date > maxSelectableDate || endOfDay(_date) < minSelectableDate;`` will cover all the cases, so we don't need to use ``endOfDay`` everywhere.

Reply:
Doing this made it an absolute point comparison.

Reply:
but why do we need ``absolute point comparison``, by the way it is good for understanding if all the dates points same time movement, but it costs three extra calls of ``endOfDay ``, which is not required.

Reply:
Got it, I will remove rest of the calls of `endOfDay` function.